README.md

```diff
@@ -1,239 +1,22 @@
-# Example TYPO3 extension for code quality checks and automated tests
```

Comment:
Please let's keep the main heading the first thing in the document (above the badges).